recipes-core/system-api-fake-fifo/system-api-fake-fifo.bb

```diff
@@ -1,16 +1,8 @@
 DESCRIPTION = "Create a regular file to accumulate system events before System API starts"
 LICENSE = "CLOSED"
-FILESEXTRAPATHS:prepend := "${THISDIR}:"
-
-SRC_URI += "file://init"
-
-INITSCRIPT_NAME = "system-api-fake-fifo"
-INITSCRIPT_PARAMS = "defaults 60"
-
-inherit update-rc.d
 
 do_install() {
-    install -m 0755 -d ${D}/var/volatile
+    install -m 1777 -d ${D}/var/volatile
```

Comment:
why change the permission settings to 1777?

Reply:
It is how this directory appears on the running instance: `drwxrwxrwt`. `t` here is a sticky bit that is set with `1` in the mode.